_docker/app/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM php:8.3.6-fpm
```

Comment:
the best way is use alpine images. They have lower size and there aren't excess dependencies in it